### PR TITLE
Update animals table with new data, queries for the questions

### PR DESCRIPTION
**Pull request by Berkay-akbas**
## Hello Code Reviewer 👋 

# Changes
- Populated animals table with the new data.
- Created a new column species. Updated the animals table by setting the species column to digimon for all animals that have a name ending in mon and for the rest column set to pokemon. Screenshot of the result:
species
- Deleted the animals table within a transaction and reverted back. Screenshot of the result:
Delete and rollback
- Deleted all animals born after Jan 1st, 2022 created a savepoint, updated all animals weight by multiplying with -1, and rolled back to the savepoint.
- Updated all animals wieghts that are negative by multiplying with -1. Screenshot of the result:
UPDATE WEIGHT
- Wrote the queries to answer questions in the project requirements. Screen shot of the result:
answers to the questions

Thank you for the review!

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -6,5 +6,6 @@ name VARCHAR(250),
 date_of_birth DATE,
 escape_attempts INT,
 neutered BOOLEAN,
-weight_kg DECIMAL
+weight_kg DECIMAL,
+species TEXT,
```
Comment (gemmen29) [inline schema.sql:10]: - [ ] Kindly update the query after rollback in that transaction to meet the [requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md#:~:text=Update%20all%20animals%27%20weights%20that%20are%20negative%20to%20be%20their%20weight%20multiplied%20by%20%2D1.). They asked only to update weights that have a negative value, so you need to use `where` clause.

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @Berkay-akbas, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql [hunk 2]
```diff
@@ -8,3 +8,31 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
...
+DELETE FROM animals;
+ROLLBACK;
+
+BEGIN;
+DELETE FROM animals WHERE date_of_birth > '2022-01-01';
+SAVEPOINT SP1;
+UPDATE animals SET weight_kg = weight_kg * -1;
+ROLLBACK TO SP1;
+UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg < 0;
+COMMIT;
```
[the remark this answers is not in the document]
Comment (gemmen29) [reply queries.sql:31]: Hello @Berkay-akbas sorry for this mistake from my side I updated the comment now.

**Review by KayLemba — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

Hello  @Berkay-akbas      , 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

HIGHLIGHTS:

- Your project is professional 
- Your repo is professional 
- Well structured files 
- No linter errors 


Now you can merge this branch and move on :1st_place_medal: 
Happy Coding cheers 🥇 💯 

well-1

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._# STATUS: APPROVED :green_circle: :clap: 

Hello  @KareemWilson     , 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

HIGHLIGHTS:

- Your project is professional 
- Your repo is professional 
- Well structured files 
- No linter errors 


Now you can merge this branch and move on :1st_place_medal: 
Happy Coding cheers 🥇 💯 

well-1

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**